nomenklatura/tui/app.py

```diff
@@ -60,26 +68,135 @@ def load(self) -> bool:
     def decide(self, judgement: Judgement) -> None:
         if self.left is not None and self.left.id is not None:
             if self.right is not None and self.right.id is not None:
+                # Hold on to pre-merge entities to show in history.
```

Comment:
```suggestion
                # Since we don't have an unresolved store as well as the resolved one,
                # hold on to pre-merge entities to show in history.
```